dms_field/models/base.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Creu Blanca
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
```

Comment:
I think we should change this to a mixin, and on each subsequent module, inherit from that mixin. This way, we avoid an extra field `dms_directory_ids` on each of the existing models in our DB. Do you agree @victoralmau @etobella ?

Reply:
I do agree. It would be cleaner

Reply:
I do agree too.